run.go

```diff
@@ -46,6 +46,15 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+func getEnv(key, fallback string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return fallback
+}
+
+var cniDir = getEnv("CNI_DIR", gocni.DefaultCNIDir)
```

Comment:
I feel this should be CNI_BIN_DIR, and should be also defined as --cni-bin-dir flag in main.go

Reply:
Also, could you sign off the commit with real name (git commit -a -s --amend), thanks

Reply:
I should have looked at the cli setup first :facepalm:

Reply:
Is there any particular order to the flags?